sdk/core/Azure.Core.TestFramework/src/ClientTestFixtureAttribute.cs

```diff
@@ -25,7 +25,18 @@ public class ClientTestFixtureAttribute : NUnitAttribute, IFixtureBuilder2, IPre
 
         private readonly object[] _additionalParameters;
         private readonly object[] _serviceVersions;
-        private readonly int? _maxServiceVersion;
+        private int? _actualPlaybackServiceVersion;
+        private int? _actualLiveServiceVersion;
+
+        /// <summary>
+        /// Specifies which service version is run during recording/playback runs.
+        /// </summary>
+        public object RecordingServiceVersion { get; set; }
+
+        /// <summary>
+        /// Specifies which service version is run during live runs.
+        /// </summary>
+        public object LiveServiceVersion { get; set; }
```

Comment:
Here are my 2 cents.
- We have disabled multi-version live testing in storage pipeline "temporarily" with a wish to enable it weekly runs (which hasn't happened yet) or pursue other solution (below). This won't work well if we want to bring back multi-version testing but cap max version and use something higher for just recording/playback. Maybe this should be `MaxLiveServiceVersion` or become collection (i.e. the Fixture attribute would look like `[ClientTestFixture(RecordingServiceVersion = ServiceVersion.V3, LiveServiceVersion = [ServiceVersion.V1, ServiceVersion.V2]`
- There's an idea in flight (see [here](https://github.com/Azure/azure-sdk/issues/2025#issuecomment-730802013)) to move service version to "test matrix generation" and effectively farm it out to more agents, so that run doesn't get longer and longer over time. When that lands the current form of `LiveServiceVersion` might better suit that (plus ability to read that from environment).

Reply:
Yeah, having it as a collection might work.